### DISCUSSION TITLE
Russian font

### DISCUSSION THREAD

**Discussion by olegkaras2** [Bodmer/TFT_eSPI#1619] [Q&A - Fonts] [answered]
Do you happen to have a Russian font?

**Comment by podaen** [comment, marked as answer]
Have you tried this?

https://github.com/Bodmer/TFT_eSPI#anti-aliased-fonts

**Reply by olegkaras2** [reply]
I can't do it that way. Do you have step by step instructions or a video example?

**Comment by shoretz** [comment]
Латиница + кириллица   https://disk.yandex.ru/d/Rv_dyB-UfC9zPQ
